src/queries/queries.js

```diff
@@ -60,9 +47,7 @@ export const buildTimeserieQueryById = timeserieId => ({
   }
 })
 
-// TODO Query used to create the aggregates of the analysis page.
-// This is a first non-optimized version. It is not guaranteed to work with a lot of data.
-// A better approach would be to use a service or a connector to pre-compute aggregation.
+// TODO replace UNSAFE_noLimit by a queryAll when https://github.com/cozy/cozy-client/issues/931 is fixed
```

Comment:
Why removing the previous TODO? This looks still relevant, as querying all the data is not really scalable

Reply:
Oui mais c'est parti en carte trello, et ce n'était pas sur la bonne query. L'info n'est pas perdue